### ISSUE TITLE
Test coverage page shows 0 for tickets where not all tickets are outlined in the list

### ISSUE THREAD

**Issue by SimiDrinkova** [testomatio/app#750]
**Describe the bug**
Test coverage page shows 0 for tickets where not all tickets are outlined in the list even if I have test cases linked with stories. Im also not able to see the stories, which we have in Jira. Even if they are open or done. 

**To Reproduce**
1. link some testcases to story 
2. go to jira
3. click Test Coverage

**Expected behavior**
I should be able to see my test coverage, when I have linked test cases to stories. Also all the stories and bugs in Jira - our project 

**Screenshots**

![Screenshot 2023-07-07 at 09 59 29](https://github.com/testomatio/app/assets/99390287/d434754b-bf2e-4fd5-b35d-d4446d5f13e8)

**Desktop (please complete the following information):**
Mac, Chrome

**Additional context**
As we spoke on the call

**Comment by TetianaKhomenko** [comment]
Add copy JQL

**Comment by TetianaKhomenko** [comment]
Hi @SimiDrinkova! We tried to investigate this issue and here is the result. We did not find the reason for the problem on our side. We added a JQL search to Test Coverage as a workaround. If this error occurs again, try finding your issue using this search. This feature will be published in the upcoming release.

![Image](https://github.com/testomatio/app/assets/77803888/bb703aec-ee06-4183-bbbd-a0bc1c707b57)
